E2E/Auto App Sync/ScreenshotTest.js

```diff
@@ -0,0 +1,376 @@
+const puppeteer = require ('puppeteer');
```

Comment:
You can leave this in the PR but I dont think we need the png files of the results.